### PR TITLE
Adds a new page called Network Impact.

### PR DESCRIPTION
**Pull request by johnnymatthews**
cc: @bobdubois

### CODE REVIEW

**Review by bobdubois — COMMENTED** [review]
Ready for review

File: content/en/storage-provider/filecoin-economics/block-rewards/index.md [hunk 1]
```diff
@@ -14,8 +12,35 @@ menu:
...
 
 The election mechanism of the Filecoin network is based on the “storage power” of the storage providers. A minimum of 10TiB in storage power is required to be eligible for WinningPoSt, and hence to earn block rewards. The more storage power a storage provider has, the more likely they will be elected to mine a block. This concept becomes incredibly advantageous in the context of [Filecoin Plus verified deals](https://docs.filecoin.io/basics/how-storage-works/filecoin-plus/).
 
+## Filecoin's storage capacity
+
+The Filecoin network is composed of storage providers who offer storage capacity to the network. This capacity is used to secure the network, as it takes a significant amount of storage to take part in the consensus mechanism. This large capacity makes it impossible for a single party to reach 51% of the network power. Therefore, it is important that the raw capacity also referred to as _raw byte power_, remains high. The Filecoin spec also included a _baseline power_ above which the network yields maximum returns for the storage providers.
```
Comment (johnnymatthews) [inline content/en/storage-provider/filecoin-economics/block-rewards/index.md:24]: > This large capacity makes it impossible for a single party to reach 51% of the network power.

Is this strictly true? It is **impossible**, or just **very hard**?

**Comment by bobdubois** [comment]
@johnnymatthews , can you merge into main please?

**Comment by ElPaisano** [comment]
@bobdubois looks like this was not ready and merged into `staging` before the Wednesday 5/31 merge window, which means that it won't be merged into `production` until this Wednesday 6/7 merge, and won't be visible on the site until then. See https://github.com/filecoin-project/filecoin-docs#merge-process for context

**Pull request merged**